test.js

```diff
@@ -25,3 +20,11 @@ test.serial('POST /api/new fails without content', async t => {
 
   t.is(res.status, 400);
 });
+
+test.serial('POST /api/new succeeds when a job is specified', async t => {
+  const res = await request('http://localhost:4041')
```

Comment:
BTW I was hoping to do `request(bootstrapApp())` or `request(app)` but somehow I was getting "app.address is not a function" errors and went on with using URL directly here. 🤷‍♂️

Reply:
I'm not familiar with `supertest`, so I'm not really getting what you mean 😆

Reply:
@simison I'll fix that up later.